README.md

```diff
@@ -1,119 +1,109 @@
 # Table of Contents:
- - [Home Chef Technical Interview Philosophy](#Home-Chef-Tech-Candidate-Interview-Process) 
- - [Home Chef Tech Candidate Interview Process](#Home-Chef-Technical-Interview-Philosophy)
- - [Candidate Technical Interview Prep Guide](#Candidate-Technical-Interview-Prep-Guide)
- - [Candidate Technical Interview Setup Guide](#Candidate-Technical-Interview-Setup-Guide)
-    - [Mac Interview Set Up Instructions Only](#Mac-Setup) **TODO BEFORE INTERVIEW**
-    - [Windows Interview Set Up Instructions Only](#Windows-Setup) **TODO BEFORE INTERVIEW**
+ - [The Technical Interview Process](#Home-Chef-Tech-Candidate-Interview-Process)
+ - [Our Interview Philosophy](#Home-Chef-Technical-Interview-Philosophy)
+ - [Interview Prep Guide](#Candidate-Technical-Interview-Prep-Guide)
+ - [Interview Setup Guide](#Candidate-Technical-Interview-Setup-Guide)
+    - [Mac Setup Instructions](#Mac-Setup) **TO DO BEFORE INTERVIEW**
+    - [Windows Setup Instructions](#Windows-Setup) **TO DO BEFORE INTERVIEW**
  - [Helpful Debugging Commands](#Helpful-Debugging-Commands)
 
 ---
 
-***Context:** This repo is designed for candidate who are currently in process with Home Chef. If you stumbled across this interview and are curious about Home Chef and think interviewing with us might be fun please reach out to `hr@homechef.com`*
+*This repo is designed for candidates who are currently in process with Home Chef. If you stumbled across this guide and think interviewing with us might be fun, please reach out to `hr@homechef.com`.*
 
-# Home Chef Tech Candidate Interview Process:
+---
+
+# The Technical Interview Process
 
-Our goal is to ensure you have a great candidate experience while moving through our interview process. On average our process takes two weeks to complete. Occasionally steps are adjusted/added/removed but typically the interview is as follows:
+Our goal is to ensure you have a great candidate experience while moving through our interview process. On average, our process takes two weeks to complete. Occasionally steps are adjusted/added/removed, but typically the interview is as follows:
 
- 1. Phone call with Recruiter or Engineering Manager - `5. hours`
- 2. Behavioral interview with 2 Home Chef engineers - `1.5 hours`
- 3. Technical interview pairing with 2 Home Chef engineers on a ticket we've completed in our production code base recently - `2 hours`
- 4. Hire or no hire decision is made within `~72 hours` of the technical interview
+ 1. Phone call with recruiter or engineering manager - .5 hours
```

Comment:
I added the `highlights` around the time just so they would stand out a bit more in the readme. Any reason in particular to remove them?

Reply:
Just stylistically I didn't think we needed them, but it's up to you 🤷 Do note the change from `5. hours` to `.5 hours` regardless.